docker/docker-compose.yaml

```diff
@@ -87,8 +87,8 @@ services:
         - ISAACSIM_ROOT_PATH_ARG=${DOCKER_ISAACSIM_ROOT_PATH}
         - ISAACLAB_PATH_ARG=${DOCKER_ISAACLAB_PATH}
         - DOCKER_USER_HOME_ARG=${DOCKER_USER_HOME}
-    image: isaac-lab-base
-    container_name: isaac-lab-base
+    image: isaac-lab-base${DOCKER_NAME_SUFFIX}
```

Comment:
Essentially, here and also in the ros2 dockerfile itself there would need to be extra logic about the hyphen if the env variable doesn't contain the hyphen by this point, which is cumbersome in these files.  I think it's better if the env variable can just already have the hyphen so that logic doesn't have to exist.

Reply:
So that the DOCKER_NAME_SUFFIX env variable is either completely empty in which case this resolves to just isaac-lab-base, or it contains the hyphen (which was inserted by python, after the user gives the non-hyphen version as input, before the environment variable was set) and also the suffix string.  So DOCKER_NAME_SUFFIX would indeed have "-custom" or "".